pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -863,13 +870,21 @@ const storeSyslogTemplateOld = `{{- define "storeSyslogOld" -}}
 const storeSyslogTemplate = `{{- define "storeSyslog" -}}
 <store>
 	@type remote_syslog
+	{{if .Hints.Has "pick_daemon_name" -}}
+	@id {{.StoreID}}_journal
```

Comment:
func StoreID() https://github.com/openshift/cluster-logging-operator/blob/b2ff008f366e01658f77a083c49a235bb2ee5e36/pkg/generators/forwarding/fluentd/fluent_conf.go#L146-L152  is intended to encapsulate the logic that  currently lives in this template as if/else statements.  Please move them there.  This additionally makes it easier to write a unit test that is not dependent upon a long string comparison

Reply:
Thank you for your review. You are correct. I fixed it.